contracts/utils/hardhat-helpers.js

```diff
@@ -5,14 +5,19 @@ const isFork = process.env.FORK === "true";
 const isArbitrumFork = process.env.FORK_NETWORK_NAME === "arbitrumOne";
 const isHoleskyFork = process.env.FORK_NETWORK_NAME === "holesky";
 const isHolesky = process.env.NETWORK_NAME === "holesky";
+const isBase = process.env.NETWORK_NAME === "base";
```

Comment:
checking chainId would be much better here

Reply:
Yup agreed, much safer